src/index.js

```diff
@@ -145,6 +159,10 @@ class Searchbase {
 
   sortOptions: Array<SortOption>;
 
+  //mic fields
+  _micStatus: MicStatusField;
```

Comment:
Can you follow the code pattern?
Put it below where we have all the private methods described, and it'll be good if you can describe the code a little bit with comments.